config/routes.rb

```diff
@@ -22,6 +22,16 @@ def draw_routes(prefix)
         end
       end
 
+      unless Rails.env.production?
+        scope 'v2', module: 'v2', as: 'v2' do
+          resources :security_guides, only: [:index, :show] do
+            resources :profiles, only: [:index, :show]
+            resources :rules, only: [:index, :show]
+            resources :value_definitions, only: :index
```

Comment:
How these routes are part of the scope? I think they should be removed from this PR.